deploy/charts/cert-manager/templates/webhook-mutating-webhook.yaml

```diff
@@ -25,7 +25,7 @@ webhooks:
           - CREATE
           - UPDATE
         resources:
-          - "*/*"
+          - "certificaterequests"
```

Comment:
I think this is a good improvement - regardless of the issue that motivates this PR. But I wonder if there is a particular reason for having a "catch-all" mutating webhook? When designing operators I would try to avoid using webhooks as much as possible - since it adds a significant operational overhead. So was this catch-all configuration intentional? Or is it leftovers? I took a look at the code, at it seems like we only have defined one mutating plugin, for certificaterequests.

I think you also should remove `acme.cert-manager.io` from the apiGroups in the webhook configuration, as that group has no certificaterequest resource, ref.  https://cert-manager.io/docs/reference/api-docs/#acme.cert-manager.io%2fv1.

Reply:
Great suggestion thank you.